swift/Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aac2324ba35cdd5631cb37618cd483887bab9cfd
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.1
```

Comment:
Ah, this is also annoying me each time I commit.
Do you think we really need to update this?

Reply:
Reverted this.